source/retryable-writes/tests/README.rst

```diff
@@ -27,7 +27,9 @@ The tests depend on a server fail point, ``onPrimaryTransactionalWrite``, which
 allows us to force a network error before the server would return a write result
 to the client. The fail point also allows control whether the server will
 successfully commit the write via its ``failBeforeCommitExceptionCode`` option.
-The fail point is described in `SERVER-29606`_.
+Keep in mind that the fail point only triggers for transaction writes (i.e. write
+commands including txnId and lsid fields).
```

Comment:
I misspoke in chat, but the field is txnNumber. Please also wrap both in double back-ticks for monospace formatting.